<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTreeView" name="treeView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Изменить</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="action_open"/>
   <addaction name="separator"/>
   <addaction name="action_save"/>
   <addaction name="separator"/>
   <addaction name="action_edit"/>
   <addaction name="action_del"/>
   <addaction name="separator"/>
   <addaction name="action_art"/>
   <addaction name="action_alb"/>
   <addaction name="action_song"/>
   <addaction name="separator"/>
   <addaction name="action"/>
   <addaction name="separator"/>
  </widget>
  <action name="action_open">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="action_edit">
   <property name="text">
    <string>Изменить</string>
   </property>
   <property name="toolTip">
    <string>Изменить</string>
   </property>
  </action>
  <action name="action_del">
   <property name="text">
    <string>Удалить</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </action>
  <action name="action_art">
   <property name="text">
    <string>Добавить Исполнителя</string>
   </property>
  </action>
  <action name="action_alb">
   <property name="text">
    <string>Добавить Альбом</string>
   </property>
  </action>
  <action name="action_song">
   <property name="text">
    <string>Добавить Песню</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Информация</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
